microk8s-cstor-operator.yaml

```diff
@@ -5386,7 +5386,7 @@ spec:
       #hostPID: true
       containers:
         - name: node-disk-manager
-          image: openebs/node-disk-manager:1.5.0
+          image: openebs/node-disk-manager:1.6.0
           args:
             - -v=4
             # The feature-gate is used to enable the new UUID algorithm.
```

Comment:
The NDM daemonset and operator parts are not updated.

Reply:
done